torchgen/model.py

```diff
@@ -1137,8 +1137,8 @@ def kind(self) -> SchemaKind:
         modifies the self argument inplace; an out schema writes
         the result into an explicitly provided out argument.
         """
-        is_inplace = self.name.name.inplace
         is_out = bool(self.arguments.out)
+        is_inplace = self.name.name.inplace
```

Comment:
does this... actually do anything?

Reply:
looks like it's just used below:
```
        if is_inplace:
            return SchemaKind.inplace
```

but agreed it feels unnecessary